### PR TITLE
Better positioning of badges

### PR DESCRIPTION
**Pull request by ngotchac**
Less intrusive badges positioning by only adding the image next to the detected email/handle/etc.
The badges and AccountCard are added at the root of the document, and are positioned correctly when needed.
Fixes some hidden overflow issues on some website (eg. Facebook)

Also puts the ETH tokens in first position.

### CODE REVIEW

File: content/components/augmentedIcon.js [hunk 1]
```diff
@@ -53,54 +55,65 @@ export default class AugmentedIcon extends Component {
...
-    const { badgesStyle, hover, open } = this.state;
+    const { badgesStyle, containerStyle, hover, open } = this.state;
 
-    const iconClasses = [styles.iconContainer];
+    const iconClasses = [ styles.iconContainer ];
+    const containerClasses = [ styles.icons ];
 
     if (hover) {
       iconClasses.push(styles.hover);
+      containerClasses.push(styles.hover);
```
Comment (tomusdrw) [inline content/components/augmentedIcon.js]: Might be better to use https://github.com/JedWatson/classnames

File: content/components/augmentedIcon.js [hunk 2]
```diff
@@ -53,54 +55,65 @@ export default class AugmentedIcon extends Component {
...
-          open={ open }
+          className={ styles.icon }
+          ref={ this.handleIconRef }
           size={ height }
-          tokens={ tokens }
-
-          onClose={ this.handleClose }
         />
+
+        <Portal into='body'>
```
Comment (tomusdrw) [inline content/components/augmentedIcon.js:85]: Pretty neat! :)

**Review by tomusdrw — APPROVED** [review]
Looks good, mustntgrumble :)

File: content/components/augmentedIcon.js (hunk 1 above)
Comment (ngotchac) [reply content/components/augmentedIcon.js]: :+1:

**Comment by ngotchac** [comment]
@tomusdrw Updated to use `classnames` module!

**Review by tomusdrw — APPROVED** [review]

**Pull request merged**